### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by nataligavryliak** [vercel/community#3734] [Account Issue] [answered]
### Question

My vercel account was locked for no reason.
User account: nataligavryliak

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Providing additional information about your account and the project you are deploying can help accelerate the account verification process.
